### PR TITLE
Add Serializer

### PR DESCRIPTION
**Pull request by Chiiizzzy**
Signed-off-by: Kaiyuan Hu <kaiyuan.hu@zilliz.com>

### CODE REVIEW

File: towhee/utils/np_format.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class NumpyArrayEncoder(json.JSONEncoder):
     """
     def default(self, o):
         if isinstance(o, np.ndarray):
-            return _NumpyFormat(o).to_dict()
+            return NumpyFormat(o).to_dict()
```
Comment (junjiejiangjjj) [inline towhee/utils/np_format.py:26]: NumpyArrayEncoder and NumpyArrayDecoder may be useless.

Comment (Chiiizzzy) [reply towhee/utils/np_format.py:26]: I'll delete them

File: towhee/utils/serializer.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# limitations under the License.
+import json
+
+import numpy as np
+from towhee.runtime.data_queue import Empty
+from towhee.utils.np_format import NumpyFormat
+from towhee.utils.empty_format import EmptyFormat
+
+
+class TowheeEncoder(json.JSONEncoder):
```
Comment (junjiejiangjjj) [inline towhee/utils/serializer.py:22]: Use the name TritonSerializer and TritonParser

File: towhee/serve/triton/pipeline_client.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 import json
 import numpy as np
 
-from towhee.utils.np_format import NumpyArrayDecoder, NumpyArrayEncoder
+from towhee.utils.serializer import TowheeDecoder, TowheeEncoder
```
Comment (junjiejiangjjj) [inline towhee/serve/triton/pipeline_client.py:17]: The client uses the new serializer,  and the server also needs it.

**Pull request merged**